eng/pipelines/common/global-build-job.yml

```diff
@@ -160,6 +173,15 @@ jobs:
         displayName: Disk Usage after Build
         condition: always()
 
+    - ${{ if eq(parameters.compilerName, 'gcc') }}:
+      - script: $(Build.SourcesDirectory)$(dir)build$(scriptExt) clr+mono+libs.native+host.native $(compilerArg) -arch ${{ parameters.archType }} -c ${{ parameters.buildConfig }} $(_officialBuildParameter) -ci
+        displayName: Build clr+mono+libs.native+host.native
```

Comment:
We should specify the subsets and compiler via the buildArgs parameter to the job instead of hard-coding it in global-build-job.yml: https://github.com/dotnet/runtime/blob/1c7100b0c21a2af147362c7b5c61408611743ded/eng/pipelines/global-build.yml#L122

Reply:
I don’t see how the other option is not equally (and even more) hard-coded.

Reply:
The other option isn’t in the job itself. It’s in the individual usage of the job template. I don’t think we should hard-code subsets in this job template itself.